src/builder/commands/copy.rs

```diff
@@ -30,9 +33,31 @@ impl Copy {
         .member("path", V::Directory::new().is_absolute(true))
         .member("ignore_regex", V::Scalar::new().default(
             r#"(^|/)\.(git|hg|svn|vagga)($|/)|~$|\.bak$|\.orig$|^#.*#$"#))
+        .member("include_regex", V::Scalar::new().optional())
         .member("owner_uid", V::Numeric::new().min(0).optional())
         .member("owner_gid", V::Numeric::new().min(0).optional())
     }
+
+    fn compile_ignore_regex(&self) -> Result<Option<Regex>, RegexError> {
```

Comment:
I would return a single structure e.g. `Filter`. which is then passed as a single structure to `check_path`.